src/AEOS/AEOSSyncTool/Adapters/ISmartFaceDataAdapter.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+
+namespace Innovatrics.SmartFace.Integrations.AEOSSync
+{
+    public interface ISmartFaceDataAdapter
+    {
+        Task<IList <SmartFaceMember>> getEmployees();
+        Task<bool> createEmployee(SmartFaceMember member, string AeosWatchlistId);
+        Task<bool> updateEmployee(SmartFaceMember member);
+        Task<bool> removeEmployee(SmartFaceMember member);
+        Task<string> initializeWatchlist();
```

Comment:
Method names should start with UpperCase
https://github.com/ktaranov/naming-convention/blob/master/C%23%20Coding%20Standards%20and%20Naming%20Conventions.md